openquake/server/local_settings.py.aelo

```diff
@@ -8,8 +8,10 @@ APPLICATION_MODE = 'aelo'
 # WEBUI config uncomment and set properly if needed
 # WEBUIURL = 'http://localhost:8800/'
 
+# TODO: ask Antonio
```

Comment:
@vot4anto may you check these USE_X_FORWARDED_HOST and USE_X_FORWARDED_PORT, please?

Reply:
We think that these variables might/should help when used in combination with NGINX, but when tested things did not behave as expected.

Reply:
This variable is not needed since we already setup the log in right format: https://github.com/gem/oq-engine/pull/8684.